bin/kitchen

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
Please get the latest version from the `template-formula`, which is `rubocop` certified!